ocaml/xapi-cli-server/records.ml

```diff
@@ -1430,6 +1430,20 @@ let pool_record rpc session_id pool =
       ; make_field ~name:"repository-proxy-username"
           ~get:(fun () -> (x ()).API.pool_repository_proxy_username)
           ()
+      ; make_field ~name:"https-only"
+          ~get:(fun () ->
+            Client.Host.get_all ~rpc ~session_id
```

Comment:
Why isn't a `Client.Pool.get_https_only` enough here? I would rather expect that a Host.set_http_only = false would make the pool field to become false as well.